pkg/reconciler/cloudeventingress/resources/service.go

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resources
+
+import (
+	"fmt"
+
+	"github.com/knative/eventing-sources/pkg/apis/sources/v1alpha1"
+	sv1alpha1 "github.com/knative/serving/pkg/apis/serving/v1alpha1"
```

Comment:
the standard way to import this would be `servingv1alpha1`, more wordy but very clear what it brings.

Reply:
Updated.